addons/l10n_be/data/account_tax_template_data.xml

```diff
@@ -1,2000 +1,3265 @@
+            <field name="invoice_repartition_line_ids" eval="[(5, 0, 0),
+                (0,0, {
+                    'factor_percent': 100,
+                    'repartition_type': 'base',
+                    'plus_tag_ids': [ref('tax_report_tag_83')],
+                }),
+
+                (0,0, {
+                    'factor_percent': 100,
+                    'repartition_type': 'tax',
+                    'account_id': ref('a411059'),
+                    'plus_tag_ids': [ref('tax_report_tag_59')],
+                }),
+            ]"/>
+            <field name="refund_repartition_line_ids" eval="[(5, 0, 0),
+                (0,0, {
+                    'factor_percent': 100,
+                    'repartition_type': 'base',
+                    'minus_tag_ids': [ref('tax_report_tag_83')],
+                    'plus_tag_ids': [ref('tax_report_tag_85')],
+                }),
+
+                (0,0, {
+                    'factor_percent': 100,
+                    'repartition_type': 'tax',
+                    'account_id': ref('a411059'),
+                    'plus_tag_ids': [ref('tax_report_tag_63')],
+                }),
+            ]"/>
         </record>
+
         <record id="attn_VAT-IN-V83-00" model="account.tax.template">
             <field name="sequence">340</field>
             <field name="description">TVA 0%</field>
             <field name="name">0% Biens d'investissement</field>
-            <field name="refund_account_id" ref="a411059"/>
-            <field name="account_id" ref="a411059"/>
             <field name="price_include" eval="0"/>
             <field name="amount">0</field>
             <field name="amount_type">percent</field>
             <field name="type_tax_use">purchase</field>
             <field name="active" eval="False"/>
             <field name="chart_template_id" ref="l10nbe_chart_template"/>
-            <field name="tag_ids" eval="[(6,0,[ref('tax_tag_83'), ref('tax_tag_85')])]"/>
             <field name="tax_group_id" ref="tax_group_tva_0"/>
+            <field name="invoice_repartition_line_ids" eval="[(5, 0, 0),
+                (0,0, {
+                    'factor_percent': 100,
+                    'repartition_type': 'base',
+                    'plus_tag_ids': [ref('tax_report_tag_83')],
+                }),
+
+                (0,0, {
+                    'factor_percent': 100,
+                    'repartition_type': 'tax',
+                    'account_id': ref('a411059'),
+                }),
+            ]"/>
+            <field name="refund_repartition_line_ids" eval="[(5, 0, 0),
+                (0,0, {
+                    'factor_percent': 100,
+                    'repartition_type': 'base',
+                    'minus_tag_ids': [ref('tax_report_tag_83')],
+                    'plus_tag_ids': [ref('tax_report_tag_85')],
+                }),
+
+                (0,0, {
+                    'factor_percent': 100,
+                    'repartition_type': 'tax
```

Comment:
Fixed in v10 here https://github.com/odoo/odoo/pull/33047 
Though, are you 100% sure? It would mean that the tag "85 not deductible" is not used at all anymore in the taxes we define in l10n_be

Reply:
nope, I'm not sure... just saying it's not what I can see in this document. We need to double check that!

Actually, the more I think about it and the more I find it strange to process the non-deductible part differently from the base in refunds (in comparison with the invoices)

@avw-odoo can you check that? what are the grids (+amounts) that should be impacted by a credit note with  a VAT of 21%, 50% deductible (car)?